src/runmode-af-packet.c

```diff
@@ -216,11 +216,40 @@ void *ParseAFPConfig(const char *iface)
         }
     }
 
-    (void)ConfGetChildValueBoolWithDefault(if_root, if_default, "use-mmap", (int *)&boolval);
+    if (ConfGetChildValueBoolWithDefault(if_root, if_default, "use-mmap", (int *)&boolval) == 1) {
+        if (boolval) {
+            aconf->flags |= AFP_RING_MODE;
+        } else {
+            SCLogInfo("Disabling mmaped capture on iface %s",
+                    aconf->iface);
+        }
+    } else {
+        aconf->flags |= AFP_RING_MODE;
+    }
+
+    (void)ConfGetChildValueBoolWithDefault(if_root, if_default, "mmap-locked", (int *)&boolval);
```

Comment:
It will enable a info message that will confuse users: SCLogInfo("Enabling locked memory for mmap on iface %s",
They will think mmap is enabled even if it's not.